washee_web/ml_electricity/controller.py

```diff
@@ -0,0 +1,41 @@
+import pandas
+from preprocessing.preprocessor import Preprocessor
+from nordpool_API.Nordpool import NordpoolAPI
+from machine_learning.machine_learning import NordpoolML
+
+class MIcontroller:
+
+    def __init__(self):
+        self.ml = NordpoolML()
+        self.preprocessor = Preprocessor()
+
+    def update_data(self):
+        path = "/Elspot/Elspot_prices/Denmark/Denmark_West"
+        file = "odedkk22.sdv"
+        
+        nordpool = NordpoolAPI()
+        nordpool.ftp_retrieve(path, file)
+        data = self.preprocessor.prune_data()
+        data = self.preprocessor.reshape_Hour_on_day_prediction(data)
+        self.preprocessor.save_data(data)
+
+    def train(self):
+        data = self.preprocessor.get_data()
+        self.ml.split(data)
+        self.ml.train()
+
+
+    def predict(self, data):
+        return self.ml.predict(data)
+
+    # def score(self):
+    #     return self.model.score()
+
+if __name__ == '__main__':
+    controller = MIcontroller()
+    controller.update_data()
+    
+    controller.train()
+    print(controller.predict([[1], [2]]))
+    print(controller.ml.score())
```

Comment:
Jeg tænker at der er brug for at splitte train og predict skarpt op i to forskellige arbejdsgange.
Man ønsker vel ikke ALTID at træne en model før man predicted på den, så går ideen med en trænet model lidt af i min optik??? men jeg kan tage fejl